packages/react-jss/src/createUseStyles.test.js

```diff
@@ -1,16 +1,72 @@
 /* eslint-disable react/prop-types */
-import createUseStyles from './createUseStyles'
-import createBasicTests from '../test-utils/createBasicTests'
 
-const createStyledComponent = (styles, options) => {
-  const useStyles = createUseStyles(styles, options)
-  const Comp = props => {
-    useStyles(props)
-    return null
-  }
-  return Comp
+import expect from 'expect.js'
+import React from 'react'
+import TestRenderer from 'react-test-renderer'
+import {stripIndent} from 'common-tags'
+
+import {SheetsRegistry, JssProvider, ThemeProvider, createUseStyles} from '.'
+
+const createGenerateId = () => {
+  let counter = 0
+  return rule => `${rule.key}-${counter++}`
+}
+
+const theme: Object = {
+  background: 'yellow',
+  background2: 'red'
 }
 
 describe('React-JSS: createUseStyles', () => {
-  createBasicTests({createStyledComponent})
+  it('should render multiple elements with applied media query', () => {
```

Comment:
I wonder if this test should be applied to both hook and hoc based implementation, just to be sure both work same way

Reply:
Can you pls elaborate, like exactly which components are needed to be tested.

Reply:
createBasicTests() is the function that runs all the tests which use the passed function which creates a styled component. You picked the test file that only serves as an entry point to generate tests using hooks interface,

If we want to run the test with both interfaces, you need to add the test inside createBasicTests

Also it seems like you removed createBasicTests() call entirely from this file, which means you don't run all the tests using hooks interface at all now